pkg/collector/corechecks/snmp/internal/report/report_device_metadata.go

```diff
@@ -228,6 +230,31 @@ func getProfileVersion(config *checkconfig.CheckConfig) uint64 {
 	return profileVersion
 }
 
+func getDeviceType(store *metadata.Store) string {
+	deviceType := strings.ToLower(store.GetScalarAsString("device.type"))
```

Comment:
I think you need to add the new field to this list: https://github.com/DataDog/datadog-agent/blob/c42154932ea0b31f418bb5e55f7e46f0ef68952f/pkg/collector/corechecks/snmp/internal/checkconfig/config_validate_enrich.go#L15-L28

Reply:
addressed in [f14ca10](https://github.com/DataDog/datadog-agent/pull/21791/commits/f14ca1045f043dae18f7581bdf1c9b4c863c643f)